service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -110,6 +111,31 @@ private void run482(CaseDetails caseDetails) {
         updateDocumentsSentToParties(caseDetails, caseData, docIds);
     }
 
+    private void run551(CaseDetails caseDetails) {
+        CaseData caseData = getCaseData(caseDetails);
+        var caseId = caseData.getId();
+        var expectedCaseId = 1641312631808724L;
```

Comment:
the case id in the [ticket](https://tools.hmcts.net/jira/browse/DFPL-551) is different. it should be 1644420520106477